snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/AllSnomedApiTests.java

```diff
@@ -63,6 +63,7 @@
 	// High-level issue test cases, Java API test cases
 	IssueSO2503RemoteJobDynamicMappingFix.class,
 	Issue3019FixDeletionOfReferringMembersTest.class,
+	CISServiceIssuesTests.class,
```

Comment:
I suggest renaming this class to something like `SnomedRF2ImportIDManagementTest` (other test class name suggestions are welcome).